File: morningsnacks/PerfectNumber.java
import java.util.Scanner;
    public class PerfectNumber{
        public static void main(String[] args){
    
        Scanner input = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int number = input.nextInt();

        int sum = 0;

        for (int count = 1; count < number; count++){
            if (number % count == 0){
                sum += count;
                }
            }

        if (sum == number){
            System.out.println(number + " is a perfect number");
            }
        else {
            System.out.println(number + " is not a perfect number");
            }
        }
        
    }

File: morningsnacks/perfect_number.py
number = int(input('Enter a number: '))

sum = 0;

for count in range (1, number):
    if (number % count == 0):
        sum += count

if (sum == number):
    print(number, "is a perfect number")
else:
    print(number, "is not a perfect number")
